src/components/Header/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
```

Comment:
Why did you choose to name the file `index.tsx` instead of `Header.tsx`?

Reply:
I used index.tsx just not to be redundant in your import statement. But I am happy to change it as Header.tsx. Let me know if you want me to change it to index.tsx as before please :)

Reply:
No, you don't have to change it. We were just curious as to why? 

Nice work!

Reply:
Thank you! 👍🏽